### DISCUSSION TITLE
[Help] Hover over row in the table - color of the row

### DISCUSSION THREAD

**Discussion by ghost** [fomantic/Fomantic-UI#1648] [Collections (form, grid, menu, message, table, ...)] [answered]
# Help Wanted

Hello. I got selectable table and when I hover over the row it changes the color to some kind of gray and also changes the mouse cursor. The behavior I am looking for is that the hower only changes the mouse cursor but do not changes the color of the row. Is this possible? BTW: what is the default color of the row when the mouse cursor hover over the selectable table?

## Testcase
Basically I am using this table https://fomantic-ui.com/collections/table.html#selectable-row. And when I hover over the row I want to change the cursor but not the color of the row.

**Comment by exoego** [comment, marked as answer]
The background color of a hovered row on a selectable row is defined here:
https://github.com/fomantic/Fomantic-UI/blob/72257802e46f7eec3ce6c68a0519026dced695e2/src/themes/default/collections/table.variables#L169
And the variable is used here:
https://github.com/fomantic/Fomantic-UI/blob/72257802e46f7eec3ce6c68a0519026dced695e2/src/definitions/collections/table.less#L622-L626

So, you can customize the background color 😉
To keep the background color, the variable should be `transparent`, I think.
If customizatoion is first time for you, the below theming page may help.
https://fomantic-ui.com/usage/theming.html

**Comment by lubber-de** [comment]
If you don't want/can build fomantic yourself , you can simply override this in additional custom CSS for the selector @exoego pointed out above
```css
.ui.ui.selectable.table>tbody>tr:hover,
.ui.table tbody tr td.selectable:hover {
  background: transparent;
}
```
See https://jsfiddle.net/lubber/ro7hsja8/1/

Closing because solution is explained
